testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/SinglenodeLdapAndFile.java

```diff
@@ -59,17 +59,7 @@ public void extendEnvironment(Environment.Builder builder)
                             CONTAINER_PRESTO_ETC + "/password.db");
         });
 
-        builder.configureContainer(TESTS, dockerContainer -> {
-            String temptoConfig = "/docker/presto-product-tests/conf/tempto/tempto-configuration-for-singlenode_ldap_and_file.yaml";
-            dockerContainer
-                    .withCopyFileToContainer(
-                            forHostPath(configDir.getPath("tempto-configuration.yaml")),
-                            temptoConfig)
-                    .withEnv("TEMPTO_CONFIG_FILES", temptoConfigFiles ->
```

Comment:
Why this env uses TEMPTO_CONFIG_FILES env, while eg MulinodeKafka binds directly to `tempto-configuration-profile-config-file.yaml`?